src/Microsoft.ML.TensorFlow/TensorFlowModel.cs

```diff
@@ -56,13 +56,6 @@ public DataViewSchema GetInputSchema()
         /// <param name="outputColumnName">The name of the requested model output.</param>
         /// <param name="addBatchDimensionInput">Add a batch dimension to the input e.g. input = [224, 224, 3] => [-1, 224, 224, 3].
         /// This parameter is used to deal with models that have unknown shape but the internal operators in the model require data to have batch dimension as well.</param>
-        /// <example>
-        /// <format type="text/markdown">
-        /// <![CDATA[
-        /// [!code-csharp[ScoreTensorFlowModel](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlowTransform.cs)]
-        /// ]]>
-        /// </format>
-        /// </example>
```

Comment:
please check with ZeeshanA. TextClassification.cs seems to be the correct sample for this API.

Reply:
/cc @zeahmed

Reply:
Yes but it is more array of input/output column and this API is just for single input/output column pair.